### PR TITLE
Added status and note fields, created migration.

### PR DESCRIPTION
**Pull request by nemodjuric**
Added 'open', 'in progress' and 'closed' status for now. Set default value to 'open'. 
Also, added 'note' text field.

### CODE REVIEW

File: app/models/feedback.rb
Comment (amprokop) [inline app/models/feedback.rb]: Elsewhere in the Caseflow world, we've opted to not use `ActiveRecord::Enum` this way, since it's a bit tricky to change later on. Instead, we usually declare enums like:

```
  enum status: {
    open: 0,
    in_progress: 1,
    closed: 2,
  }
```
See https://bendyworks.com/blog/rails-enum-sharp-knife for an explanation of why.
And see `caseflow-efolder/app/models/download` for an example of this approach. Can we use that here?

File: db/migrate/20161128153630_add_details_to_feedback.rb
Comment (amprokop) [inline db/migrate/20161128153630_add_details_to_feedback.rb]: how about `admin_note`? that way it's clearer what this is

**Review by amprokop — CHANGES_REQUESTED** [review]

File: db/migrate/20161128153630_add_details_to_feedback.rb
Comment (nemodjuric) [inline db/migrate/20161128153630_add_details_to_feedback.rb]: I've changed it to admin_note and added integers to enum.

**Comment by amprokop** [comment]
LGTM after merging from master and after addressing my comments on created_at and updated_at

**Pull request merged**
